cmake/CMakeLists.txt

```diff
@@ -181,6 +182,16 @@ if(onnxruntime_ENABLE_LTO)
     endif()
 endif()
 
+if(onnxruntime_DISABLE_RTTI)
+  if(MSVC)
```

Comment:
Can you add an explicit option to build.py to disable rtti? 

Builds are generally done using that and it's easier for a user that way vs. having to discover onnxruntime_DISABLE_RTTI in this file and specifying it via --cmake_extra_defines)